spec/rubocop/cop/rails/file_path_spec.rb

```diff
@@ -56,4 +56,13 @@
       expect(cop.offenses.size).to eq(1)
     end
   end
+
+  context 'Rails.root.join used with / operator' do
```

Comment:
The description here is not clear enough. Obviously this pertains to having used `/` in the context of interpolation.